docs/symptom-survey/data-access.md

```diff
@@ -12,11 +12,16 @@ spread of COVID-19 and its effects on public health and well-being. This may
 help improve our local and national responses to the pandemic and our
 understanding of how it has affected society.
 
-De-identified data can be made available to researchers associated with
+De-identified individual survey responses can be made available to researchers associated with
 universities or non-profit organizations. To request access to the data please
 submit the information requested in [Facebook's page on obtaining data
 access](https://dataforgood.fb.com/docs/covid-19-symptom-survey-request-for-data-access/),
 which sets out the basic conditions and provides a form to request access. An
 [international version of the COVID Symptom Survey](https://covidmap.umd.edu/)
 is conducted by the University of Maryland (UMD) and access can be requested
 through the same form.
+
+High-level aggregates of select survey items are available in the [COVIDcast
+API](../api/fb-survey.md), more information [here](../api/covidcast.md). Finer aggregates grouped by
+various demographic characteristics are available for download, more information
+[here](./contingency-tables.md).
```

Comment:
drive-by comment -- if we want people with screen readers to be able to use our data, we should make sure the link text is descriptive. Links often just get listed out all together after the body text of the page has been read, and nobody will be able figure out which instance of "here" goes to which document.

Relevant W3C criteria:
* https://www.w3.org/TR/WCAG21/#link-purpose-in-context
* https://www.w3.org/TR/WCAG21/#link-purpose-link-only